### PR TITLE
Python 3.9 support

### PR DESCRIPTION
**Pull request by guedou**
This PR adds Python 3.9 support for Github Actions.

Note: for Travis, the latest version is 3.8, and AppVeyor is 3.7.

### CODE REVIEW

File: .github/workflows/unittests.yml [hunk 1]
```diff
@@ -12,7 +12,7 @@ jobs:
       - name: Setup Python
         uses: actions/setup-python@v1
         with:
-          python-version: 3.8
+          python-version: 3.9
```
Comment (gpotter2) [inline .github/workflows/unittests.yml:15]: They actually released 3.9 for github a few hours after your commit. Could you use the v2 version of the package though?

Comment (guedou) [reply .github/workflows/unittests.yml:15]: Done!

File: .github/workflows/unittests.yml [hunk 2]
```diff
@@ -56,12 +56,12 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python: [2.7, pypy2, pypy3, 3.5, 3.6, 3.7, 3.8]
+        python: [2.7, pypy2, pypy3, 3.5, 3.6, 3.7, 3.8, 3.9]
```
Comment (gpotter2) [inline .github/workflows/unittests.yml:59]: What do you think about removing 3.5 from here? It also went EOL but unlike 2.7 it's barely used.

Comment (guedou) [reply .github/workflows/unittests.yml:59]: I will rather keep it unless there is a feature that we cannot implement.

**Comment by guedou** [comment]
Could this PR be merged?

**Review by gpotter2 — APPROVED** [review]

**Comment by gpotter2** [comment]
I just realized this included a temporary vim file @guedou

**Comment by guedou** [comment]
I just pushed a PR that removes the file.

**Pull request merged**